### DISCUSSION TITLE
Caching not working properly

### DISCUSSION THREAD

**Discussion by TheoOliveira** [LazyVim/LazyVim#316] [Q&A]
I've installed few days ago on NIxOS and I've noticed when change tabs and they were recently saved or not save yet and it gets back gives a warning say there is an existing swap file already and everytime i try or to override it mess up the file with combining things i've edited already and other stuff from previous savings.

**Comment by folke** [comment]
Can you be more specific what the issue is here? And how is this related to the cache? If you have the same file open or didn't save and force quit Neovim, then you will always get the swap warning

**Reply by TheoOliveira** [reply]
That's the thing. I didn't quit nvim, I just switched back to the previous tab.

**Reply by TheoOliveira** [reply]
BTW, I am using tmux. What info you do you need for you to try to reproduce? I think it's related in this context when i use tmux .
